request-contracts/Dockerfile

```
@@ -3,7 +3,7 @@ ARG TAG=master
 
 FROM node:${NODE_VERSION} AS base
 WORKDIR /app
-RUN apk add --no-cache git
+RUN apk add --no-cache git python3 make build-base
```

`build-base` contains `make`, see https://pkgs.alpinelinux.org/package/v3.3/main/x86/build-base "depends" table.
This should work:
```suggestion
RUN apk add --no-cache git python3 build-base
```